### PR TITLE
TT-186

### PR DESCRIPTION
**Pull request by MontgomeryWatts**
Privacy status for new charges is a radio

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -86,32 +86,42 @@ export default {
 /* Create a custom checkbox */
   .checkmark {
     position: absolute;
-    top: 15;
+    top: 0;
```
Comment (MontgomeryWatts) [inline src/App.vue]: This property wasn't actually doing anything since it omitted 'px'. Because of this changing it to 0 has no effect.

Comment (lukasyelle) [reply src/App.vue]: Good catch!

File: src/components/CommitteeAdmin.vue [hunk 2]
```diff
@@ -123,12 +130,12 @@ export default {
     closeAddNewCharge () {
       this.createChargeTitle = null
       this.createChargeDescription = null
-      this.createChargePriority = null
+      this.createChargePriority = 1
```
Comment (MontgomeryWatts) [inline src/components/CommitteeAdmin.vue]: If you try to create more than one charge in a row, you get an error that you can't fix without refreshing the page. This sets createChargePriority to a valid value upon closing the modal.

File: src/components/CommitteeAdmin.vue [hunk 3]
```diff
@@ -106,7 +113,7 @@ export default {
       createChargePriority: 1,
       createChargeDescription: null,
       createChargePawLink: null,
-      isPrivate: null,
+      isPrivate: true,
```
Comment (MontgomeryWatts) [inline src/components/CommitteeAdmin.vue]: I changed this from null to true for clarity on the front end.

File: src/components/CommitteeAdmin.vue (hunk 2 above)
Comment (elijah-parrish) [reply src/components/CommitteeAdmin.vue]: Good catch!

**Pull request merged**